packages/file_selector/file_selector/example/lib/get_directory_page.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:file_selector/file_selector.dart';
+import 'package:flutter/material.dart';
+
+/// Screen that shows an example of openFile(s)
+class GetDirectoryPage extends StatelessWidget {
+  void _getDirectoryPath(BuildContext context) async {
+    final String initialDirectory = '~/Desktop';
```

Comment:
Do the various OS file dialogs actually expand "~"? The robust way to do this would be to use `path_provider`; if we don't want to depend on that I would suggest we just don't set a starting directory in the example.

Reply:
That is a good point. I'm not sure if `~` gets actually expanded so I will play safe here and remove it. 
Thanks @stuartmorgan :)!

Reply:
Note that `path_provider` doesn't work in web at all AFAIK (U_U)